### PR TITLE
fix(tlsn-server-fixture): update cert fixtures

### PR DESCRIPTION
**Pull request by sinui0**
This PR updates our expired cert fixtures, a temporary measure until we replace them with certs generated at runtime.

### CODE REVIEW

File: components/tls/tls-server-fixture/src/root_ca.der
Comment (yuroitaki) [inline components/tls/tls-server-fixture/src/root_ca.der:1]: I used to be confused by this der file, whether it's the private key or the cert, would it be clearer if it's named `root_ca_cert.der`?

File: components/tls/tls-server-fixture/src/test_server.der
Comment (yuroitaki) [inline components/tls/tls-server-fixture/src/test_server.der:1]: same as https://github.com/tlsnotary/tlsn/pull/499/files#r1637442633

**Review by yuroitaki — COMMENTED** [review]
LGTM, only one comment

**Review by th4s — APPROVED** [review]
lgtm

**Pull request merged**